dev-haskell/cabal/cabal-3.2.1.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+#hackport: flags: -parsec-struct-diff
+
+CABAL_FEATURES="lib profile test-suite"
+CABAL_FEATURES+=" nocabaldep" # in case installed Cabal is broken
+inherit haskell-cabal
+
+MY_PN="Cabal"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A framework for packaging Haskell software"
+HOMEPAGE="https://www.haskell.org/cabal/"
+SRC_URI="https://hackage.haskell.org/package/${MY_P}/${MY_P}.tar.gz"
+
+PATCHES="${FILESDIR}/${PN}-3.2.1.0-disable-hackage-test.patch"
```

Comment:
This commit does not add the file. I would normally expect `RESTRICT=test` and removal of `CABAL_FEATURES=test-suite` to be a way to disable tests. It would probably avoid you to change depends below.

Reply:
This patch is intended to remove a *single* test, while still allowing the rest of the test-suite to execute.

I will dig into this commit and fix it, I probably messed something up when I rebased prior to submitting the PR.

Reply:
@trofi the missing file should be in there now.

Reply:
Actually, scratch that, I removed the testing altogether so this is no longer needed.

Testing was taken out due to the "circular dependency" thing.